app-libs/stf/Cargo.toml

```diff
@@ -39,6 +39,8 @@ sp-runtime = { default-features = false, git = "https://github.com/paritytech/su
 my-node-runtime = { package = "integritee-node-runtime", optional = true, git = "https://github.com/integritee-network/integritee-node.git", branch = "master" }
 pallet-parentchain = { default-features = false, git = "https://github.com/integritee-network/pallets.git", branch = "master" }
 
+# oli dependencies
+poc = { path = "src/poc" }
```

Comment:
It is not good practice to declare other crates in the source folder, you should move it one level above. Inside the source folder you should only declare modules.